stm32-modules/heater-shaker/firmware/motor_task/motor_policy.hpp

```diff
@@ -35,6 +36,8 @@ class MotorPolicy {
 
   private:
     static constexpr uint16_t MAX_SOLENOID_CURRENT_MA = 330;
-    int32_t ramp_rate = DEFAULT_RAMP_RATE_RPM_PER_S;
+    double ramp_rate_rpm_per_ms =
```

Comment:
why did we convert this to ms?

Reply:
Because it actually always has been whoops. The function in the MCLib we're calling, [MCI_ExecSpeedRamp](https://github.com/Opentrons/opentrons-modules/blob/652bbbc848feceba61801114bbc078f6cf341e66/stm32-modules/heater-shaker/firmware/motor_task/mc_interface.h#L85-L86), has its duration set in ms, and I just... never noticed until now.